pkg/release/unpacker.go

```diff
@@ -97,12 +99,18 @@ func Unpack(ctx context.Context, s *Staged) (*Unpacked, error) {
 		return nil, err
 	}
 
+	ctlBinaries, err := unpackCtlBinariesFromRelease(ctx, s)
+	if err != nil {
+		return nil, err
+	}
+
 	log.Printf("Extracted %d component bundles from images archive", len(bundles))
 	return &Unpacked{
 		ReleaseVersion:        s.Metadata().ReleaseVersion,
 		GitCommitRef:          s.Metadata().GitCommitRef,
 		YAMLs:                 yamls,
 		Charts:                charts,
+		CtlBinaries:           ctlBinaries,
```

Comment:
Hm, I wonder if we're better off re-categorising these different fields slightly in future.. e.g. `charts`, `github assets` and `images` or something. This is fine for now, but is quite specific to cert-manager's `ctl` binary. What if we had a 'migration-tool' or something in future? It would need to be treated the same as this, but would be an awkward fit :)

Reply:
We could certainly do that in the future